### PR TITLE
LPS-155035

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Forwarded from https://github.com/liferay-lima/liferay-portal/pull/3178

> ## Motivation
> 
> https://issues.liferay.com/browse/LPS-155035
> ## Proposed solution
> 
> Preserve the language direction from the current request language rather than use the language direction of the language itself.
> ## Steps to verify
> 
> Please see linked LPS.

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:reopen

**Comment by drewbrokke** [comment]
The forward was a mistake, my apologies.

**Comment by holatuwol** [comment]
@drewbrokke

**Comment by drewbrokke** [comment]
@holatuwol ah so we are just talking about the field label, not the field itself?

File: modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit_plo_entry.jsp [hunk 1]
```diff
@@ -141,7 +141,7 @@ renderResponse.setTitle(editDisplayContext.getPageTitle());
 							String languageId = LanguageUtil.getLanguageId(availableLocale);
 						%>
 
-							<div class="form-group" dir="<%= LanguageUtil.get(availableLocale, "lang.dir") %>" lang="<%= LocaleUtil.toW3cLanguageId(availableLocale) %>">
+							<div class="form-group" dir="<%= LanguageUtil.get(request, "lang.dir") %>" lang="<%= LocaleUtil.toW3cLanguageId(availableLocale) %>">
 								<aui:input label="<%= TextFormatter.format(languageId, TextFormatter.O) %>" name='<%= "value_" + availableLocale %>' value="<%= valuesLocalizedValuesMap.get(availableLocale) %>" wrappedField="<%= true %>" />
```
Comment (drewbrokke) [inline modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit_plo_entry.jsp:145]: Let's add the original `dir` attribute statement to the `aui:input`. That way, the label will follow the user's locale, but the input itself will still honor the translation language's direction.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by ambrinchaudhary** [comment]
Hi all, sorry for not replying here before. I just "forwarded" this pull that landed on lima queue but I guess we cannot help much here, as it is not our field of expertise.

**Comment by drewbrokke** [comment]
@ambrinchaudhary got it, thanks!

See https://github.com/liferay-usm/liferay-portal/pull/638

**Pull request closed without merging**